content/news/playtest-20180729.md

```diff
@@ -0,0 +1,70 @@
+Hot off the presses, we are pleased to announce a new OpenRA playtest series. With your help, we hope to find and squash any lurking issues and produce a refined release version in a couple of months time.
+
+The leading feature in playtest-20180729 is the integration of player accounts using our new [OpenRA Forums](https://forum.openra.net/). This optional feature allows players to:
+
+* Confirm their identity to other players without revealing your IP address or location
+* Connect to private clan- or tournament-specific servers without using passwords
+* Add some flair by displaying badges that show their achievements or preferences
+
+Anonymous players continue to be identified by their IP and geo-location.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180729-profiles.png" width="600" alt="Player Profiles">
```

Comment:
This isn't using the normal "!\[\<text>](\<link>)" notation (because of width=600?), although I'm not sure if we want to enforce that.

Reply:
> (because of width=600?)

This is a hi-dpi image, so the standard markdown syntax breaks things.